resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterScheduler.scala

```diff
@@ -710,7 +734,7 @@ private[spark] class MesosClusterScheduler(
   }
 
   private def copyBuffer(
-      buffer: ArrayBuffer[MesosDriverDescription]): ArrayBuffer[MesosDriverDescription] = {
+      buffer: TraversableOnce[MesosDriverDescription]): ArrayBuffer[MesosDriverDescription] = {
```

Comment:
Can we avoid this change?

Reply:
No. This is due to the `queuedDrivers` collection whose type has changed from `ArrayBuffer` to `mutable.TreeSet` (with an `Ordering` for sorting the drivers following their priority).

Reply:
Hi, @pgillet . Usually, you had better keep the comment open for the review. The commenters will check your reply and resolve by themselves.

Reply:
Sorry about that.